astropy/coordinates/tests/accuracy/test_fk4_no_e_fk4.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
 
 from astropy import units as u
-from astropy.coordinates.angle_utilities import angular_separation
+from astropy.coordinates import angular_separation
 from astropy.coordinates.builtin_frames import FK4, FK4NoETerms
```

Comment:
Shouldn't we be merging these from `astropy.coordinates` too? (And also in other test files).

Reply:
Yes. But PR creep.

Reply:
@eerovaher - I noticed that too, but ended up thinking it was out of scope here. A cleanup PR might be called for...

Reply:
Yes, I agree this should be dealt with in a separate pull request. I suspect `coordinates` is not the only sub-package that requires cleanup. I'll look into how wide-spread this is in `astropy`.

Reply:
It's quite! I've done a little cleanup in cosmology, but I don't think it's finished there either.